src/main/java/rx/Completable.java

```diff
@@ -39,66 +38,76 @@
 @Experimental
 public class Completable {
     /** The actual subscription action. */
-    private final CompletableOnSubscribe onSubscribe;
+    private final OnSubscribe onSubscribe;
+
+    /**
+     * @deprecated Use {@link OnSubscribe} instead.
+     */
+    @Deprecated // TODO Remove interface after 1.2.0 release. It was never a stable API. Provided only for migration.
+    public interface CompletableOnSubscribe extends OnSubscribe {
+
+    }
+
     /**
      * Callback used for building deferred computations that takes a CompletableSubscriber.
      */
-    public interface CompletableOnSubscribe extends Action1<CompletableSubscriber> {
+    public interface OnSubscribe extends Action1<rx.CompletableSubscriber> {
         
     }
+
+    /**
+     * @deprecated Use {@link Operator} instead.
+     */
+    @Deprecated // TODO Remove interface after 1.2.0 release. It was never a stable API. Provided only for migration.
+    public interface CompletableOperator extends Operator {
+
+    }
     
     /**
      * Convenience interface and callback used by the lift operator that given a child CompletableSubscriber,
      * return a parent CompletableSubscriber that does any kind of lifecycle-related transformations.
      */
-    public interface CompletableOperator extends Func1<CompletableSubscriber, CompletableSubscriber> {
+    public interface Operator extends Func1<rx.CompletableSubscriber, rx.CompletableSubscriber> {
```

Comment:
I'm not certain about this being a compatible change.

So does this still compile?

```java
CompletableOperator op = new CompletableOperator() {
    @Override
    public Completable.CompletableSubscriber call(Completable.CompletableSubscriber cs) {
         return cs;
    }
}
```